garden-share-08/garden-share-fe
# Feature connect redux

#### What does  this PR do?
- Creates reducer for ProductPage
- Renders ProductPage with data
- Creates reusable Form 

#### Where should the reviewer start?
- productPage directory
- form directory

#### How should this be manually tested?
- Pull down branch and run app
- Navigate to homepage
- Click on a listing to see that listing's page

#### Any background context you want to provide?
- App is consuming mockData at this point

#### What are the relevant tickets?
- closes #14
